includes/class-wcpdf-main.php

```diff
@@ -334,7 +334,7 @@ public function generate_pdf_ajax() {
 			// Check the user privileges
 			$full_permission = WPO_WCPDF()->admin->user_can_manage_document( $document_type );
 			if ( ! $full_permission ) {
-				if ( ! isset( $_GET['my-account'] ) ) {
+				if ( ! isset( $_GET['my-account'] ) || ! isset( $_GET['shortcode'] ) ) {
```

Comment:
Right now you are breaking both the my account action button and the shortcode download link. One of these conditional statements will always return `true` because either `my-account` isn't set or `shortcode` isn't set. You will have to use `&&` instead of `||` here.

Reply:
Yep, my bad.

Reply:
No problem. I also updated the comment two lines down.